docs/docusaurus/docs/guides/validation/checkpoints/how_to_pass_an_in_memory_dataframe_to_a_checkpoint.md

```diff
@@ -1,27 +1,28 @@
 ---
-title: Pass an in-memory DataFrame to a Checkpoint
+title: Validate Checkpoints with Expectations and data
```

Comment:
I think it would be more accurate to say "Validate data with Expectations and Checkpoints."  But the guide we would actually want in place of this one is: "Validate data and Expectations passed to a Checkpoint at runtime."

Reply:
I think we should make a note to rework this document fully.  As it stands, the process can be applied to more than just in-memory data.

Reply:
@Rachel-Reverie I have updated the topic title as suggested. Who would be the SME that could help with the content rework? Do you prefer to take it on? If so, should we leave this PR open?